tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorDataType.spec.ts

```diff
@@ -0,0 +1,339 @@
+    await page.locator('[title="#c90076"] >> nth=1').click();
+    await page.locator('.sp-choose >> nth=1').click();
+    // Adds a thumbnail
+    await page.locator('[key="blockEditor_addThumbnail"]').click();
+    await page.locator('[data-element="tree-item-wwwroot"]').locator('[data-element="tree-item-expand"]').click();
+    await page.locator('[data-element="tree-item-media"]').locator('[data-element="tree-item-expand"]').click();
+    await page.locator('[data-element="tree-item-' + imagePath + '"]').locator('[data-element="tree-item-expand"]').click();
+    await page.locator('.umb-tree-item__label', {hasText: fileName}).click();
+
+    await page.locator('[label-key="buttons_submitChanges"]').click();
+
+    // Updates properties in the block list editor
+    await page.locator('[name="numberFieldMin"]').fill('1');
+    await page.locator('[name="numberFieldMax"]').fill('10');
+    await page.locator('[id="useLiveEditing"]').click();
+    await page.locator('[id="useInlineEditingAsDefault"]').click();
+    await page.locator('[id="maxPropertyWidth"]').fill('100px');
+
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.save));
+
+    // Assert
+    await expect(page.locator('.alert-success', {hasText: 'Datatype saved'})).toBeVisible();
+
+    // Clean
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementNameTwo);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+    await umbracoApi.stylesheets.ensureNameNotExists(stylesheetName + '.css');
+    await umbracoApi.media.ensureNameNotExists(imageName);
+  });
+
+  test('can delete a block list editor', async ({page, umbracoApi, umbracoUi}) => {
+    const elementNameTwo = 'AnotherTestElement';
+
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementNameTwo);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+
+    const element = await createElement(umbracoApi, elementName);
+    const elementTwo = await createElement(umbracoApi, elementNameTwo);
+
+    const blockListType = new BlockListDataTypeBuilder()
+      .withName(blockListName)
+      .addBlock()
+        .withContentElementTypeKey(element['key'])
+      .done()
+      .addBlock()
+        .withContentElementTypeKey(elementTwo['key'])
+      .done()
+      .build();
+    await umbracoApi.dataTypes.save(blockListType);
+
+    // Navigates to the block list editor
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+    await page.locator('[data-element="tree-item-dataTypes"]').locator('[data-element="tree-item-expand"]').click();
+    await page.locator('[data-element="tree-item-' + blockListName + '"]').click({button: "right"});
+
+    // Deletes the block list editor
+    await page.locator('[data-element="action-delete"]').click();
```

Comment:
We should use the new `clickDataElementByElementName` in the UI helper 👍 
and remember to use Constants again with actions 👍